packages/react-components/playwright.config.components.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import { defineConfig, devices } from '@playwright/experimental-ct-react';
+import react from '@vitejs/plugin-react';
+import type { TestOptions } from './tests/browser/FlavoredBaseTest';
+import {
+  ANDROID_USER_AGENT,
+  DESKTOP_16_TO_9_VIEWPORT,
+  DESKTOP_4_TO_3_VIEWPORT,
+  DESKTOP_USER_AGENT,
+  MINUTE,
+  chromeLaunchOptions
+} from '../../common/config/playwright/playwrightConfigConstants';
+
+const notBetaBuildVariable = process.env['COMMUNICATION_REACT_FLAVOR'] !== 'beta';
```

Comment:
nit: avoid variable suffix, and negative is perhaps unecessary but thats just opinion

```suggestion
const isBetaBuild = process.env['COMMUNICATION_REACT_FLAVOR'] === 'beta';
```